templates/locales/en.yaml

```diff
@@ -26,7 +26,7 @@ PatientPasswordResetHello: "Hello,"
 PatientPasswordResetRequest: "You requested a password reset."
 PatientPasswordResetDidNotRequest: "If you didn't request this, please ignore this email."
 PatientPasswordResetByDevice: "Resetting your password has to be done on your Diabeloop device."
-PatientPasswordResetExplainKey: "Follow the instructions with the below key on on your Diabeloop device."
+PatientPasswordResetExplainKey: "Follow the instructions with the below key on your Diabeloop device."
```

Comment:
not needed as https://github.com/mdblp/hydrophone/pull/92 will rework the patient password reset labels

Reply:
We should leave it here anyway to prevent from repushing new changes